map.css

```diff
@@ -90,6 +90,24 @@ html, body, #map {
 		margin-right: 10px;
 	}
 }
+
+.panel .panel-content {
+	overflow-y: auto;
+    position: absolute;
+    width: calc(100% - 5px);
+    height: calc(100% - 10px);
+}
+.panel .panel-content .close {
```

Comment:
Why is this required? It should work as-is (provided you do not modify order of elements added).